tests/unit/amo/pages/TestLanguageTools.js

```diff
@@ -90,157 +81,129 @@ describe(__filename, () => {
+    const { store } = dispatchClientMetadata({ lang });
     store.dispatch(loadLanguageTools({ languageTools: [] }));
 
-    const root = renderShallow({ store });
+    render({ store });
 
-    expect(root.find(LoadingText)).not.toHaveLength(0);
+    expect(screen.queryAllByClassName('LoadingText').length).toBeGreaterThan(0);
   });
 
   it('renders language tools in your locale', () => {
     const { store } = dispatchClientMetadata({ lang: 'fr' });
     store.dispatch(loadLanguageTools({ languageTools }));
 
-    const root = renderShallow({ store });
+    render({ store, lang: 'fr' });
 
-    const dictionary = root.find(
-      `.LanguageTools-in-your-locale-list-item--${ADDON_TYPE_DICT}`,
-    );
-    const langPack = root.find(
-      `.LanguageTools-in-your-locale-list-item--${ADDON_TYPE_LANG}`,
+    expect(
+      screen.getByRole('heading', { name: 'Available for your locale' }),
+    ).toBeInTheDocument();
+
+    const dictionary = screen.getByClassName(
+      `LanguageTools-in-your-locale-list-item--${ADDON_TYPE_DICT}`,
     );
+    expect(
+      within(dictionary).getByText('le French Dictionary'),
+    ).toBeInTheDocument();
 
-    expect(root.find(LoadingText)).toHaveLength(0);
-    expect(root.find('.LanguageTools-in-your-locale')).toHaveLength(1);
-    expect(dictionary).toHaveLength(1);
-    expect(dictionary.find(Link)).toHaveProp(
-      'children',
-      'le French Dictionary',
+    const langPack = screen.getByClassName(
+      `LanguageTools-in-your-locale-list-item--${ADDON_TYPE_LANG}`,
     );
-    expect(langPack).toHaveLength(1);
-    expect(langPack.find(Link)).toHaveProp('children', 'French Language Pack');
+    expect(langPack).toBeInTheDocument();
+    expect(
+      within(langPack).getByText('French Language Pack'),
+    ).toBeInTheDocument();
   });
 
   it('omits "language tools in your locale" section if none available', () => {
     const { store } = dispatchClientMetadata({ lang: 'pt-BR' });
     store.dispatch(loadLanguageTools({ languageTools }));
 
-    const root = renderShallow({ store });
+    render({ store, lang: 'pt-BR' });
 
-    expect(root.find('.LanguageTools-in-your-locale')).toHaveLength(0);
+    expect(
+      screen.queryByRole('heading', { name: 'Available for your locale' }),
+    ).not.toBeInTheDocument();
   });
 
   it('renders language packs in the table view for the right language', () => {
-    const { store } = dispatchClientMetadata();
+    const { store } = dispatchClientMetadata({ lang });
     store.dispatch(loadLanguageTools({ languageTools }));
 
-    const root = renderShallow({ store });
-
-    expect(root.find('.LanguageTools-lang-en-GB')).toHaveLength(1);
-    expect(root.find('.LanguageTools-lang-en-US')).toHaveLength(1);
-    expect(root.find('.LanguageTools-lang-fr')).toHaveLength(1);
-    expect(root.find('.LanguageTools-lang-ur')).toHaveLength(1);
-    expect(root.find('.LanguageTools-lang-zh-TW')).toHaveLength(1);
-
+    render({ store });
+
+    expect(
```

Comment:
Is there something else we could be looking for, other than `className`? We try to only resort to locating by `className` when nothing else is possible. Could we not be looking for the presence of the name of each of these langpacks?